### ISSUE TITLE
TimeSelect is not being displayed properly

### ISSUE THREAD

**Issue by plazma-prizma** [Hacker0x01/react-datepicker#1116]
<img width="524" alt="screen shot 2017-11-04 at 10 48 51" src="https://user-images.githubusercontent.com/8254052/32403559-d0c973fc-c145-11e7-9b82-0982d36eadb5.png">

**Comment by daniltulin** [comment]
Experience the same problem, guys

**Comment by Rahulkr204** [comment]
I am not even getting the time section.
![screen shot 2017-11-07 at 10 23 47 am](https://user-images.githubusercontent.com/8544323/32477676-e6e6eab8-c3a5-11e7-8b6c-d76a48089dec.png)
![screen shot 2017-11-07 at 10 24 18 am](https://user-images.githubusercontent.com/8544323/32477677-e72535ca-c3a5-11e7-94a3-fa9c3f33618c.png)

**Comment by matong1745** [comment]
I got the same problem TimeSelect is not working. Even upgrade to the ver. 0.60.0

**Comment by orx0r** [comment]
It is temporary fix the issue:

1. 
```
import 'react-datepicker/dist/react-datepicker.css';
```

2. add some inline styles
```
.react-datepicker__time-container .react-datepicker__time .react-datepicker__time-box ul.react-datepicker__time-list {
    padding-left: 0;
    padding-right: 0;
}
```

**Comment by Rahulkr204** [comment]
@orx0r Added your fix to my code, still struggling with problem. Can you share your code snippet of DatePicker component used ?

**Comment by orx0r** [comment]
Hi @Rahulkr204 sure. this works in my case
```
// imports
import 'react-datepicker/dist/react-datepicker.css';

...
// component
return (
    <div>
        <style>
          {`.react-datepicker__time-container .react-datepicker__time .react-datepicker__time-box ul.react-datepicker__time-list {
            padding-left: 0;
            padding-right: 0;
          }`}
        </style>
        <DatePicker
          dateFormat="YYYY-MM-DD HH:mm"
          showTimeSelect
          timeFormat="HH:mm"
          selected={Moment(this.state.date)}
          onChange={this.onChange}
        />
    </div>
);
```

**Comment by codepandy** [comment]
Hi @orx0r 
it's invalid the code that you show

**Comment by vvbvargas** [comment]
@orx0r works like a charm, thank you!!!

**Comment by shrutic** [comment]
@orx0r Thanks!!! Works great !! The code snippet really helped

**Comment by oldmill1** [comment]
Perfect!!

**Comment by bskinny129** [comment]
The temp fix helped on desktop, but mobile looks like this:

![screen shot 2017-12-07 at 5 15 20 pm](https://user-images.githubusercontent.com/1643783/33746476-692b7b18-db72-11e7-98d4-f46807374aa5.png)

**Comment by willm** [comment]
@bskinny129 I did this to make it work on smaller screens (probably not very elegant, but it worked)
```css
.react-datepicker__time-container .react-datepicker__time .react-datepicker__time-box ul.react-datepicker__time-list {
        padding-left: unset;
        padding-right: unset;
        width: 100px;
}

.react-datepicker__input-container {
        width:100%;
}

.react-datepicker-wrapper {
        width:100%;
}

.react-datepicker {
        width: 314px;
}```

**Comment by bskinny129** [comment]
@willm Thank you, that really helped!

In case it helps anyone, here is my complete settings that I tried to optimize for mobile (doesn't pop up keyboard and is a little smaller to consistently fit on the screen):

 ```
.react-datepicker__time-container .react-datepicker__time .react-datepicker__time-box ul.react-datepicker__time-list {
                      padding-left: unset;
                      padding-right: unset;
                      width: 100px;
}
.react-datepicker__input-container, .react-datepicker-wrapper {
                      width:100%;
}
.react-datepicker {
                      width: 314px;
}
.react-datepicker .react-datepicker__day {
                line-height: 1.4rem;
                margin: .1rem 0.166rem;
}
.react-datepicker .react-datepicker__day-name {
                line-height: 1rem;
}
```
```
<DatePicker autoFocus readOnly popperPlacement="bottom-start" 
       popperModifiers={{
              offset: {
                enabled: true,
                offset: '0px, 12px'
              },
              preventOverflow: {
                enabled: true,
                escapeWithReference: false, 
                boundariesElement: 'viewport'
              }
            }}
        shouldCloseOnSelect={false} selected={this.state.tempDate} onChange={this.datePickerChanged} 
        dateFormat="M/D/YY h:mm a" timeIntervals={15} timeFormat="h:mm a" showTimeSelect />
```

**Comment by flameddd** [comment]
@orx0r and all others guys 
thanks all of you to save my days :)

**Comment by hnineitpm** [comment]
@orx0r thank you! it work now

**Comment by Mohammad-Khalid23** [comment]
@orx0r Thank you it worked for me

**Comment by dandua98** [comment]
Is there going to be a more permanent fix for this? Running `react-datepicker v1.6` with `React 15.6`.
